packages/react-tooltip/src/components/Tooltip/useTooltip.tsx

```diff
@@ -63,13 +66,12 @@ export const useTooltip = (
     resolveShorthandProps(props, tooltipShorthandProps),
   );
 
-  const theme = useTheme();
   const popper = usePopper({
     enabled: visible,
     position: state.position,
     align: state.align,
     offset: [0, state.offset + (state.noArrow ? 0 : arrowHeight)],
-    arrowPadding: theme?.global ? 2 * parseInt(tooltipBorderRadius(theme), 10) : 0,
+    arrowPadding: 2 * tooltipBorderRadius,
```

Comment:
what does `2` do ? can we mitigate using magic numbers and assign them into self describing constant ?

Reply:
The `2` is for styling purposes, and comes from the visual spec. The name of the constant here is `arrowPadding`, which is defined as twice the border radius. It is unfortunate that popper requires some style-related values to be defined here instead of `useTooltipStyles`.

I generally agree with you about not using magic numbers, but we don't typically give names to styling constants beyond what they style (in this case, `arrowPadding`). I _could_ define e.g. `const tooltipBorderRadiusMultipleForArrowPadding = 2;` but that seems fairly useless?